pyupgrade/_plugins/versioned_branches.py

```diff
@@ -124,15 +137,19 @@ def visit_If(
                     ) and
                     len(node.test.ops) == 1 and (
                         _eq(node.test, 2) or
-                        _compare_to_3(node.test, ast.Lt)
+                        _compare_to_3(node.test, ast.Lt, min_version[1]) or
+                        any(
+                            _compare_to_3(node.test, (ast.Lt, ast.LtE), minor)
+                            for minor in range(min_version[1])
+                        )
```

Comment:
I think this `any` block could be eliminated and subsumed into the `_compare_to` function

right now it's linear on the version being compared which could potentially be bad (?)

I'm going to merge this as is and we can always fix it later?

Reply:
Sounds like a plan :)